src/pages/ReimbursementAccount/BankAccountManualStep.js

```diff
@@ -119,7 +119,12 @@ class BankAccountManualStep extends React.Component {
                                 <Text>
                                     {this.props.translate('common.iAcceptThe')}
                                 </Text>
-                                <TextLink href="https://use.expensify.com/terms">
+                                <TextLink
+                                    href="https://use.expensify.com/terms"
+
+                                    // to not trigger input blur event. Because form is validating on input blur event link is moving down before click event.
```

Comment:
This is wrong, the input blur is still called. My analysis is, calling `preventDefault` in `onMouseDown` make the `onPress` in the `TextLink` call before the input blur.

How about _**Call the `onPress` in the `TextLink` before the input blur is fired and shift the link element.**_

Reply:
Yeah, you are right. My wording sounds like it blocks blur event altogether.
Just to clarify what is actually happening is blur event should be called after <code>onMouseDown</code> event. Since we are calling `preventDefault` on `onMouseDown` event blur is not triggered so `onPress` can be called without issue. Since blur event has not been called after `onMouseDown` event it gets called after `onPress` event.

Reply:
@mollfpr changed it.